test/functional/lua/glob_spec.lua

```diff
@@ -205,6 +205,16 @@ describe('glob', function()
       eq(true, match('[!a-zA-Z0-9]', '!'))
     end)
 
+    it('should handle long patterns', function()
+      local fill_200 =
+        'aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa'
+      local long_lit = fill_200 .. 'a'
```

Comment:
Perhaps
```suggestion
      eq(200, fill_200:len())
      local long_lit = fill_200 .. 'a'
```
along with a note that the LPEG max recursion depth is 200, just to be a bit clear what that's doing